resources/model/OpenStudio.idd

```diff
@@ -35642,7 +35641,7 @@ OS:LifeCycleCost:UsePriceEscalation,
         \key ElectricitySurplusSold
         \key ElectricityNet
         \key NaturalGas
-        \key Steam
+        \key DistrictHeatingSteam
```

Comment:
No "DistrictHeatingWater" here?

Reply:
E+ doesn't support it yet, I'm trying to fix that in:
* https://github.com/NREL/EnergyPlus/pull/10229

Wasn't sure if I should try to support it just yet when E+ doesn't.

Reply:
I added a specific note in https://github.com/NREL/OpenStudio/issues/4934

![image](https://github.com/NREL/OpenStudio/assets/5479063/1ab0061b-6cdb-4df2-9a87-24ed0c6b37da)